src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -470,6 +471,7 @@ private void addClassTypes(
     Set<Class<?>> visitedClasses = new LinkedHashSet<>(); // consider each class just once
     for (String classname : classnames) {
       Class<?> c = getClass(classname, errorHandler);
+      c = CoverageTracker.instance.getInstrumentedClass(c != null ? c.getName() : "");
```

Comment:
`getInstrumentedClass` returns null if its argument is `""`, but I had to look it up to learn that.  This would be clearer as
```
      c = (c != null ? CoverageTracker.instance.getInstrumentedClass(c.getName()) : null;
```
or
```
      if (c != null) {
        c = CoverageTracker.instance.getInstrumentedClass(c.getName());
      }
```